Makefile

```diff
@@ -193,14 +193,14 @@ elasticsearch-mappings:
 build-examples:
 	esc -pkg frontend -o examples/hotrod/services/frontend/gen_assets.go  -prefix examples/hotrod/services/frontend/web_assets examples/hotrod/services/frontend/web_assets
 ifeq ($(GOARCH), s390x)
-	CGO_ENABLED=0 installsuffix=cgo go build -o ./examples/hotrod/hotrod-$(GOOS)-$(GOARCH) ./examples/hotrod/main.go
+	CGO_ENABLED=0 installsuffix=cgo go build -trimpath -o ./examples/hotrod/hotrod-$(GOOS)-$(GOARCH) ./examples/hotrod/main.go
```

Comment:
it would be better if we define `GOBUILD=CGO_ENABLED=0 installsuffix=cgo go build -trimpath` in L30 and use `$(GOBUILD)`, to DRY this up.

Reply:
Makes total sense. I've updated the code.